sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,209 @@
+    )
+    assert len(ownership_list) == 0
+
+    ownership_list = []
+
+    for i in range(ownership_cnt):
+        ownership = _create_ownership(str(i), "owner_id", None, None)
+        ownership_list.append(ownership)
+    result_ownership_list = checkpoint_store.claim_ownership(ownership_list)
+    assert ownership_list != result_ownership_list
+    assert len(result_ownership_list) == len(ownership_list)
+    for i in range(len(ownership_list)):
+        assert ownership_list[i]["etag"] != result_ownership_list[i]["etag"]
+        assert (
+            ownership_list[i]["last_modified_time"]
+            != result_ownership_list[i]["last_modified_time"]
+        )
+
+    ownership_list = checkpoint_store.list_ownership(
+        fully_qualified_namespace, eventhub_name, consumer_group
+    )
+    assert len(ownership_list) == ownership_cnt
+    assert len(ownership_list) == len(result_ownership_list)
+    for i in range(len(result_ownership_list)):
+        assert ownership_list[i]["etag"] == result_ownership_list[i]["etag"]
+        assert (
+            ownership_list[i]["last_modified_time"]
+            == result_ownership_list[i]["last_modified_time"]
+        )
+
+
+def _update_and_list_checkpoint(storage_connection_str, table_name):
+    fully_qualified_namespace = "test_namespace"
+    eventhub_name = "eventhub"
+    consumer_group = "$default"
+    partition_cnt = 8
+
+    checkpoint_store = TableCheckpointStore.from_connection_string(
+        storage_connection_str, table_name
+    )
+    checkpoint_list = checkpoint_store.list_checkpoints(
+        fully_qualified_namespace, eventhub_name, consumer_group
+    )
+    assert len(checkpoint_list) == 0
+    for i in range(partition_cnt):
+        checkpoint = _create_checkpoint(i, 2, 20)
+        checkpoint_store.update_checkpoint(checkpoint)
+
+    checkpoint_list = checkpoint_store.list_checkpoints(
+        fully_qualified_namespace, eventhub_name, consumer_group
+    )
+    assert len(checkpoint_list) == partition_cnt
+    for checkpoint in checkpoint_list:
+        assert checkpoint["offset"] == "2"
+        assert checkpoint["sequence_number"] == 20
+
+    checkpoint = _create_checkpoint(0, "30", 42)
+    checkpoint_store.update_checkpoint(checkpoint)
+    checkpoint_list = checkpoint_store.list_checkpoints(
+        fully_qualified_namespace, eventhub_name, consumer_group
+    )
+    assert len(checkpoint_list) == partition_cnt
+    assert checkpoint_list[0]["offset"] == "30"
+
+
+@pytest.mark.parametrize("storage_connection_str", STORAGE_CONN_STR)
+def test_claim_ownership_exception(storage_connection_str):
+    storage_connection_str, table_name = get_live_storage_table_client(
+        storage_connection_str
+    )
+    try:
+        _claim_ownership_exception_test(storage_connection_str, table_name)
+    finally:
+        remove_live_storage_table_client(storage_connection_str, table_name)
+
+
+@pytest.mark.parametrize("storage_connection_str", STORAGE_CONN_STR)
```

Comment:
can we add the cosmos connection string and mark this as live (which will ensure that this test re-runs in the pipeline each time)?
```suggestion
@pytest.mark.parametrize("storage_connection_str", [AZURE_STORAGE_CONN_STR, AZURE_COSMOS_CONN_STR])
@pytest.mark.liveTest
```